node.js

```diff
@@ -1444,9 +1444,32 @@ async function main() {
   // syntactic sugar for arguments
   const args = minimist(process.argv.slice(2));
 
+  // kludge to deconflict node IDs from hashed values
```

Comment:
Just to clarify. You see this as a temporary stopgap until we increase the size of our hash table, right?

Reply:
Yup.  Although I'm sure we'll want to keep it around long after because it is handy.  So, I tagged it as a "kludge" currently because I'm sure there is a more elegant way to achieve this  . . . and it will likely stay a kludge since we'll like using it for debug and never did the more elegant way :-)  Or, maybe this isn't bad in which case it shouldn't be marked as a kludge.

Reply:
(in other words, i'll leave it like this in my next commit but am happy to revisit one way or the other without hurting my feelings)